l10n_ch_fds_postfinance/model/fds_postfinance_account.py

```diff
@@ -0,0 +1,238 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Swiss Postfinance File Delivery Services module for Odoo
+#    Copyright (C) 2015 Compassion CH
+#    @author: Nicolas Tran
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import models, fields, api, exceptions, _
+import logging
+import base64
+import tempfile
+import shutil
+import pysftp
+
+_logger = logging.getLogger(__name__)
+
+
+class fds_postfinance_account(models.Model):
+    ''' the FDS PostFinance configuration that allow to connect to the
+        PostFinance server
+    '''
+    _name = 'fds.postfinance.account'
+
+    name = fields.Char(
+        required=True
+    )
+    hostname = fields.Char(
+        string='SFTP Hostname',
+        default='fds.post.ch',
+        required=True,
+    )
+    postfinance_email = fields.Char(
+        default='fds@post.ch',
+        required=True,
+        help='E-mail of fds postfinance'
+    )
+    username = fields.Char(
+        string='SFTP Username',
+        required=True,
+    )
+    company_contact_id = fields.Many2one(
+        comodel_name='res.users',
+        string='Company Contact',
+        ondelete='restrict',
+        required=True,
+        help='user that have create/contact with the fds postfinance account'
+             ', it will be used to send the public key to postfinance'
+    )
+    fds_authentication_keys_ids = fields.One2many(
+        comodel_name='fds.authentication.keys',
+        inverse_name='fds_account_id',
+        string='Authentication keys',
+    )
+    fds_postfinance_files_ids = fields.One2many(
+        comodel_name='fds.postfinance.files',
+        inverse_name='fds_account_id',
+        string='FDS Postfinance files',
+        readonly=True,
+        help='downloaded files from sftp'
+    )
+    fds_PF_files_directory_ids = fields.One2many(
+        comodel_name='fds.postfinance.files.directory',
+        inverse_name='fds_account_id',
+        string='FDS postfinance directories',
+    )
+
+    ##################################
+    #         Button action          #
+    ##################################
+    @api.multi
+    def verifyDirectories_button(self):

```

Comment:
Please don't mix camel case and snake case.

BTW: in python conventions are snake_case for function name and variable
CamelCase is used for class names